src/lib.rs

```diff
@@ -93,7 +93,7 @@ use tokio::process::Command;
 
 #[tracing::instrument(level = "debug", skip_all, fields(command = %format!("{:?}", command.as_std())))]
 async fn execute_command(command: &mut Command) -> Result<Output, ActionError> {
-    tracing::trace!("Executing `{:?}`", command.as_std());
```

Comment:
Oh, because we already have `fields(command = ....)` above

Reply:
Ya I noticed it twice and was like :woman_facepalming: